common_features.mk

```diff
@@ -131,7 +131,7 @@ ifeq ($(strip $(LED_MATRIX_ENABLE)), IS31FL3731)
     OPT_DEFS += -DIS31FL3731
     COMMON_VPATH += $(DRIVER_PATH)/issi
     SRC += is31fl3731-simple.c
-    SRC += i2c_master.c
+    QUANTUM_LIB_SRC += i2c_master.c
```

Comment:
SRC and LIB_SRC are paired. Similarly, QUANTUM_SRC and QUANTUM_LIB_SRC are paired. So I think it would be better to use LIB_SRC here.